core/templates/dev/head/components/common-layout-directives/navigation-bars/side-navigation-bar.directive.html

```diff
@@ -125,7 +125,11 @@
 
     <li>
       <a href="https://medium.com/oppia-org" target="_blank" rel="noopener">
-        <img ng-src="<[$ctrl.getStaticImageUrl('/sidebar/blogger.png')]>" class="oppia-sidebar-menu-icon" alt="Oppia Blog">
+        <picture>
+          <source type="image/webp" ng-srcset="<[$ctrl.getStaticImageUrl('/sidebar/blogger.webp')]>">
+          <source type="image/jpeg" ng-srcset="<[$ctrl.getStaticImageUrl('/sidebar/blogger.png')]>">
```

Comment:
> Do we need this when `ng-src="<[$ctrl.getStaticImageUrl('/sidebar/blogger.png')` is set in line below? Also, shouldnt the `type=image/png`?

We need the `<source>` and `<img>` both  because the `<source>` is the fallback if the WebP is not supported and the `<img>` is the fallback when picture tag is not supported.

Reply:
> Ditto for the other places where `<picture>` is used.

Replacing `image/jpeg`. Sorry for the mistake.

Reply:
Thanks for explanation, makes sense.